testsuite/forge_test.py

```diff
@@ -575,15 +673,43 @@ def testMain(self) -> None:
             )
         ]))
         filesystem = SpyFilesystem({
-            "temp-comment": get_fixture_path("testMainComment.fixture").read_bytes(),
-            "temp-step-summary": get_fixture_path("testMainComment.fixture").read_bytes(),
-            "temp-pre-comment": get_fixture_path("testMainPreComment.fixture").read_bytes(),
-            "temp-report": get_fixture_path("testMainReport.fixture").read_bytes(),
+            "temp-comment": get_fixture_path(
+                "testMainComment.fixture"
+            ).read_bytes(),
+            "temp-step-summary": get_fixture_path(
+                "testMainComment.fixture"
```

Comment:
No `testStepSummary.fixture`?

Reply:
If you look at step summary it is the same as the comment